integration_test/cases/pool.exs

```diff
@@ -16,9 +16,7 @@ defmodule Ecto.Integration.PoolTest do
 
   test "can start multiple repos" do
     # Can't start a second Repo with the default name
-    {_, pool_name, _} = PoolRepo.__pool__
-    pool_pid = Process.whereis(pool_name)
-    assert {:error, {:already_started, ^pool_pid}} = PoolRepo.start_link()
+    assert {:error, {:already_started, _}} = PoolRepo.start_link()
```

Comment:
Must this point to the pool process? In this branch a supervisor (which is registered as `Module.concat(pool_name, Supervisor)`)